src/display.ts

```diff
@@ -119,6 +127,60 @@ export const determineAction = async (
 	return [action, replacer];
 };
 
+export const determineHistoryIssue = async () => {
```

Comment:
I think this function should be renamed, as well as `HistoryIssue`. Doesn't sound right to me. Maybe `PastIssue` or something?